src/Control/Carrier/Debug.hs

```diff
@@ -12,18 +12,49 @@ module Control.Carrier.Debug (
   module X,
 ) where
 
-import Control.Carrier.AtomicState
-import Control.Carrier.Diagnostics
-import Control.Carrier.Output.IO
+import Control.Carrier.AtomicState (AtomicStateC, modify, runAtomicState)
+import Control.Carrier.Diagnostics (
+  Algebra (..),
+  Diagnostics,
+  Handler,
+  Has,
+  SomeDiagnostic (..),
+  ToDiagnostic (renderDiagnostic),
+  errorBoundaryIO,
+  rethrow,
+  run,
+  send,
+  thread,
+  (~<~),
+  type (:+:) (..),
+ )
+import Control.Carrier.Output.IO (OutputC, output, runOutput)
 import Control.Carrier.Simple (Simple, sendSimple)
-import Control.Effect.Debug as X
-import Control.Effect.Lift
+import Control.Effect.Debug as X (
+  Algebra (..),
+  Debug (..),
+  Handler,
+  Has,
+  debugEffect,
+  debugError,
+  debugLog,
+  debugMetadata,
+  debugScope,
+  run,
+  send,
+  thread,
+  (~<~),
+  type (:+:) (..),
+ )
```

Comment:
Can you leave these two big expansions as implicit for now, and we'll address it separately?  We don't want to limit the re-exports with explicit lists.